SPECS/ucx/ucx.spec

```diff
@@ -88,7 +108,7 @@ Provides header files and examples for developing with UCX.
 %configure --disable-optimizations \
            %{!?debug:--disable-logging} \
            %{!?debug:--disable-debug} \
-           %{!?debug:--disable-assertions} \
+           %{!?debug:--disable-assertions --enable-mt} \
```

Comment:
Can we stick to one setting per line? I think consistency helps to avoid mistakes and makes it easier to read the build configuration.

Reply:
Fix duplicate